docs/src/content/docs/manual-setup.mdx

```diff
@@ -102,26 +102,34 @@ If you have an existing Astro project, you can use Starlight to quickly add a do
 
 ### Use Starlight at a subpath
 
-To add all Starlight pages at a subpath, place all your docs content inside a subdirectory of `src/content/docs/`.
+To add all Starlight pages at a subpath, you could relay on top of Astro.js configurations. `base` and `assetsPrefix`
```

Comment:
Does this mean you can use Starlight only on a single subpath?

I want to document more than one distinct "project", each on a different subpath, having a separate ToC, but adding up to a single shared search index. E.g.:

/site (docs describing my own website setup)
/homelab (docs describing my homelab)
/sports (docs describing my cycling gear, associated electronics and configuration)